e2e/provision/playbooks/roles/bootstrap/README.md

```diff
@@ -14,9 +14,10 @@ Available variables are listed below, along with default values (see defaults/ma
 
 | Variable                        | Required | Default                                                       | Choices                   | Comments                                                                          |
 |---------------------------------|----------|---------------------------------------------------------------|---------------------------|-----------------------------------------------------------------------------------|
-| host_min_vcpu                   | no       | 8                                                             |                           | Minimum vCPUs required                                                            |
-| host_min_cpu_ram                | no       | 16                                                            |                           | Minimum RAM required (GB)                                                         |
-| host_min_root_disk_space        | no       | 50                                                            |                           | Minimum disk space required (GB)                                                  |
+| host_reqs.sanbox.vcpu           | no       | 6                                                             |                           | Minimum vCPUs required for Sandbox installation                                   |
```

Comment:
@electrocucaracha did you verify that it takes 6 vCPU? Probably we can go even lower right? Since we are mentioning minimum.

Reply:
Yes, the Sandbox without any workload (that means only the Nephio, Cluster API, MetalLB, Cert Manager, Porch and Gitea workloads in the Management cluster), have 2600m cpu request and 5100m limit, and 2970Mi memory request and 5510Mi limit.

Those requirements can be useful when you separated the workload clusters in a separated VM(s) provided by a cloud provider.

Reply:
Okay perfect. Similary for end to end we really need 16 vCPU?